zerver/openapi/zulip.yaml

```diff
@@ -19567,6 +19567,21 @@ paths:
                         [mentions]: /help/mention-a-user-or-group
                     - $ref: "#/components/schemas/GroupSettingValue"
                   example: 11
+                can_manage_group:
+                  allOf:
+                    - description: |
+                        A [group-setting value][setting-values] defining the set of users who
+                        have permission to [manage this user group][manage-user-groups].
+
+                        This setting cannot be set to `"role:internet"` and `"role:everyone"`
```

Comment:
We haven't decided how to point to the permission config here.

Reply:
Yeah, we can clean this up in a follow-up sweep.